ui/app/components/recovery/page/header.hbs

```diff
@@ -0,0 +1,18 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+<Hds::PageHeader class="has-top-padding-l has-bottom-padding-m" as |PH|>
+  <PH.Title>
+    Secrets Recovery
+    {{#if this.version.isCommunity}}
+      <Hds::Badge @text="Enterprise" @color="highlight" data-test-badge="enterprise" />
+    {{/if}}
+  </PH.Title>
+
+  <PH.Subtitle>
+    Recover lost or deleted secrets from a raft snapshot. Supported types include KV v1 and Cubbyhole.
+  </PH.Subtitle>
+</Hds::PageHeader>
+{{! TODO if in snapshot load form, will show breadcrumbs instead }}
```

Comment:
Nice! Are you thinking that for the relevant pages we'll pass in a `@breadcrumbs` arg or something?